src/wp-includes/customize/class-wp-customize-selective-refresh.php

```diff
@@ -193,7 +193,7 @@ public function export_preview_data() {
 		);
 
 		// Export data to JS.
-		printf( '<script>var _customizePartialRefreshExports = %s;</script>', wp_json_encode( $exports ) );
+		wp_print_inline_script_tag( sprintf( /** @lang JavaScript */ 'var _customizePartialRefreshExports = %s;', wp_json_encode( $exports ) ) );
```

Comment:
PhpStorm flags the placeholder as a syntax error:

> Expression expected

![image](https://github.com/WordPress/wordpress-develop/assets/134745/2bf41d43-9374-4cd1-88ec-e7798f98f2bc)